ftw/linkchecker/test_setup.py

```diff
@@ -180,6 +181,11 @@ def set_up_test_environment(portal):
                             content['textblock_within2'],
                             content['textblock_url2'], )
 
+    # set workflow for pages
+    wftool = api.portal.get_tool('portal_workflow')
+    wftool.setChainForPortalTypes(
+        ['ftw.simplelayout.ContentPage'], 'Sl Page Workflow')
```

Comment:
I'm not sure what this does, but you don't need to explain or change anything. It's more a note to myself.

Reply:
This sets the workflow `Sl Page Workflow' for all content being `ftw.simplelayout.ContentPage`. I do that because in my tests sl content pages do not have a workflow otherwise and there would be no output to be tested.

Reply:
Your explanation would make an excellent code comment 😉

Reply:
I've just added this comment in position 😻

Reply:
Perfect.